samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -19,6 +19,9 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 #include "mfx_samples_config.h"
 
+#if defined(_WIN32) || defined(_WIN64)
+#include <windows.h>
```

Comment:
Strange placement. Could we move it somewhere to sample_common?

Reply:
Nope. It is used for particular purpose in this paticular file, no need to move it to common files